### ISSUE TITLE
StructuralEstimation.jl and friends

### ISSUE THREAD

**Issue by pkofod** [JuliaLang/METADATA.jl#7549]
I figured I would discuss this here before making PRs, as I'm bound to mess something up, if it turns out that something is not OK after I make 5 different PRs.

I want to register a package called `StructuralEstimation.jl` that depends on a few packages: `MDPTools.jl`, `NestedFixedPoint.jl`, `NestedPseudoLikelihood.jl`, and `StructuralBase.jl`. The packages do
- `MDPTools.jl` is a package used to formulate and solve Markov Decision Processes. At the moment a quite limited subset of MDPs, but it can/will be expanded.
- `StructuralBase.jl` basically provides some common methods and types for use by the individual estimation packages
- `NestedFixedPoint.jl` implements the nested fixed point (NFXP) estimation method by Rust (1987, Econometrica) for the models `StructuralEstimation.jl` aims to handle
- `NestedPseudoLikelihood.jl ` implements the nested pseudo likelihood (NPL) estimation method by Aguirregabiria and Mira (2002, Econometrica) for the same class of models
- `StructuralEstimation.jl` pulls it all together. The typical user will probably `using` this package, but either of the other packages can be used on their own (`StructuralBase.jl` being the most useless on its own as is clear from the name).

The dependencies goes like this 
![screenshot from 2017-01-12 14-31-33](https://cloud.githubusercontent.com/assets/8431156/21891699/3dfd2cbc-d8d4-11e6-83b8-4a384f959778.png)

I know naming of packages has been an issue in the past, so I just wanted to see if I could get something like a :+1: before starting the process of registering the packages. They can be registered in the order: MDPTools, NestedPseudoLikelihood, NestedFixedPoint, StructuralBase, and then StructuralEstimation. Other packages will be added alongside NestedFixedPoint and NestedPseudoLikelihood in the future, but those are not done yet.

Best,

**Comment by tkelman** [comment]
I don't love the MDP acronym, but there's already registered MDPs and POMDPs packages, and an unregistered MarkovTools package. Given the landscape and alternatives, these seem fine.

**Comment by pkofod** [comment]
> I don't love the MDP acronym, but there's already registered MDPs and POMDPs packages, and an unregistered MarkovTools package. Given the landscape and alternatives, these seem fine.

I've been over a lot of names for that one. Writing it out seems quite long though. I think it's one of those acronyms that are both widely used, and easily googleable.

**Comment by pkofod** [comment]
I could maybe do StructuralModels.jl? StructuralBase comes after in the graph, but could be called StructuralEstimationBase for all I care.

**Comment by DilumAluthge** [comment]
METADATA.jl is no longer maintained, so I am closing this issue.

If this issue is still relevant, please re-open a new issue on the Julia General registry: https://github.com/JuliaRegistries/General
